tutorial_plugin_java.md

```diff
@@ -0,0 +1,154 @@
+﻿---
+layout: page
+title: Tutorial Plugin Java
+permalink: /plugin/tutorial/java
+---
+
+# How to create a plugin for SmartSHARK
+This repository contains a minimal working example of a java plugin for SmartSHARK. This short readme is a tutorial on how to program a plugin starting from the ground. The first section will provide you with the basic information. More advanced information will be outlined for each file in the files section.
+
+## Basics
+In this section, we will shortly explain the basics of plugin development.
+### Get started
+This  is a quick start guide. 
+1. Modify the content of the setup.py and the info.json with your information
```

Comment:
There is no setup.py for Java plugins. My guess is you mean the other files in the plugin_packaging?